ext/native-decls/GetPedDecorations.md

```diff
@@ -0,0 +1,24 @@
+---
+ns: CFX
+apiset: client
+---
+## GET_PED_DECORATIONS
+
+```c
+object GET_PED_DECORATIONS(Ped ped);
+```
+
+Returns a list of decorations applied to a ped.
+
+The data returned adheres to the following layout:
+```
+[{collectionHash1, overlayHash1}, ..., {collectionHashN, overlayHashN}]
+```
+
+Only works on the invoking client. Calling it on another client results in undefined data.
```

Comment:
What would 'invoking client' mean? It's a client-side native, the client invoking the native would always be the invoking client.

I'm guessing _remote players_ won't have accurate data, instead?

For example:

> This command will return undefined data if invoked on a remote player ped.